webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -152,6 +153,26 @@ export default class SuggestionBox extends React.PureComponent {
         actions: PropTypes.shape({
             addMessageIntoHistory: PropTypes.func.isRequired,
         }).isRequired,
+
+        /**
+         * Props for input
+         */
```

Comment:
Some dependency upgraded its internal prop-types version, and hence the typing for it got strict. But its fine as we don't have much usages of it anyways left.

Reply:
As above, I think this is actually the update to `@types/react-redux`. I think it had difficulty inferring the types of propTypes before, which caused it to just end up being typed less strictly. Either that, or the old version was fine with typing existing props but the update prevents us from providing extra props which is normally fine

Reply:
lets leave these types here as is, because having a bit extra types is good thing and also this component is due to be migrated to typescript at that time we can just extend the HTMLcomponent input types.